src/content/guides/deploying-istio/index.md

```diff
@@ -0,0 +1,283 @@
+
+# Check istioctl is available
+$ istioctl version
+Version: X.X.X
+...
+BuildStatus: Clean
+```
+
+Secondly, in order to avoid a known issue with the sidecar security context (privileges), it is needed to extend or create the pod security policy tha will be assigned to your micro services. Istio, beyond the sidecar container, injects a init container in the app's deployment resource. It takes care of forwarding traffic between the proxy and the original container(s).
+```
+initContainers:
+  - args:
+    ...
+    image: docker.io/istio/proxy_init:0.6.0
+    name: istio-init
+    securityContext:
+      capabilities:
+        add:
+        - NET_ADMIN
+      privileged: true
+```
+
+As consequence, the pod now must have the right security policies configured to run in the cluster. An easy way to solve the problem is to create a pod security policy that contains the correct capabilities. Then, define an istio role and a role binding for wiring it together against the default service account for the namespace where your apps are running.
+
+__Warning:__ If your apps have a defined a customer service account then you need to create the binding for each of them.
+
+```
+apiVersion: extensions/v1beta1
+kind: PodSecurityPolicy
+metadata:
+  name: istio
+spec:
+  privileged: true
+  allowedCapabilities:
+  - 'NET_ADMIN'
+  fsGroup:
+    rule: RunAsAny
+  runAsUser:
+    rule: RunAsAny
+  seLinux:
+    rule: RunAsAny
+  supplementalGroups:
+    rule: RunAsAny
+  volumes:
+  - '*'
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: istio
+rules:
+- apiGroups:
+  - extensions
+  resourceNames:
+  - istio
+  resources:
+  - podsecuritypolicies
+  verbs:
+  - use
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: istio
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: istio
+subjects:
+- kind: ServiceAccount
+  name: default
+  namespace: <NAMESPACE_OF_YOUR_SERVICES>
+```
+
+## Recomendations 
+
+The deployment is made up of a different number of components. Some of them, as pilot, have a large impact in terms of memory and CPU, so it is recommended to have around 8GB and 4CPUs free in your cluster. Obviously, all components have `requested resources` defined, so in case you don't have enough capacity you will see pods not starting.
+
+A followed convention, not only within Istio community but Kubernetes too, is to define in all your app deployments the label `app` and `version` as metadata. Both labels help the tracing and metric system to have richer meta information. 
+
+Another good practice is to name the service ports. Istio uses the name to discover the protocol used by the end service container. Thus you have to prefix the port name with the protocol desired. The different supported protocols (http, http2, grpc, mongo, or redis) leverage Istio to route traffic more intelligently. Otherwise, it will use plain TCP routing.
```

Comment:
Does istio expect certain protocol names here? If yes, we should make that more explicit, format the example names as literals (in backticks) and probably link to a full reference.